pkgs/by-name/sp/spades/package.nix

```
@@ -1,42 +1,57 @@
 {
   lib,
   stdenv,
-  fetchurl,
-  zlib,
   bzip2,
   cmake,
+  fetchurl,
```

```suggestion
  fetchurl,
```
Is there a good reason for not using `fetchFromGitHub`? (Which is the preferable fetcher abstraction for GitHub)

---

Checking...

---

Sorry, had to rebuild as tarballs seems to differ... and tests are not passing anymore...  still checking why...

---

The file https://github.com/ablab/spades/blob/next/src/CMakeListsInternal.txt is empty in the SPAdes-4.0.0.tar.gz release file. That makes the "github" version tests failing as they probably included some tests that are to be run only from their dev environement.
I thought that the flag `-DSPADES_BUILD_INTERNAL=OFF` would disable this, but it does nothing...
So, I emptied the CMakeListsInternal.txt as they did for their release file...